docs/overrides/partials/nav-item.html

```diff
@@ -0,0 +1,283 @@
+{# This overrides the nav-item partial from mkdocs-material v9.6.17 #}
+{# https://github.com/squidfunk/mkdocs-material/blob/c89a66bf39bb7f472e925753367ff8e464e0d683/src/templates/partials/nav-item.html #}
+{# Majority of the file is the original, so our changes are highlighted #}
+{# with `OUR CHANGES START` and `OUR CHANGES END` #}
```

Comment:
As this comment says, this is an override for `mkdocs-material`, and 99% of the file remains the same, only the `render_content()` macro was modified.